content/en/certificates.md

```diff
@@ -148,50 +204,9 @@ These intermediate CAs are no longer being used to issue Subscriber certificates
   * Valid until: 2027-03-12
   * CA details: [crt.sh](https://crt.sh/?caid=295815), [issued certs](https://crt.sh/?Identity=%25&iCAID=295815)
   * Certificate details (signed by ISRG Root X1): [der](/certs/2024/r11.der), [pem](/certs/2024/r11.pem), [txt](/certs/2024/r11.txt)
-* **Let's Encrypt Authority X1**
```

Comment:
I'm not sure I agree with dropping these - I think this may still be useful for historical reference.  It's hidden away in a retired collapsed section on the website so there's not much overhead of keeping it here, yeah?

Reply:
These certificates have been expired for four years, they have not been covered by our audits for two years, and their keys are being destroyed. The certificates themselves are still available in the /static/ directory, and this content is still available in our git history. But fair enough, it's an unrelated change, I'll do this in a separate PR.